setup.py

```diff
@@ -13,38 +13,6 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import os
-import sys
+from setuptools import setup
 
-from setuptools import (
-    find_packages,
-    setup,
-)
-
-assert sys.version_info >= (3,), "Python 3 is required"
-
-
-VERSION = "0.0.1"
```

Comment:
Sorry if this is evident, I'm just wondering where you specify project ans python version

Reply:
version isn't really all that important on this project, as we always use the latest from git, but it's easy enough to add, same for python version. I just pushed a change that adds those.